scripts/deploy/gce/etc/letsencrypt/renewal-hooks/deploy/fix-permissions

```diff
@@ -0,0 +1,6 @@
+#!/usr/bin/env bash
+
```

Comment:
I think it wouldn’t hurt to add `set -euo pipefail`

Reply:
Only `-e` would make any sense here, but it doesn't matter anyway: this is invoked by `certbot` during a timer run. No-one will be looking anyway. If it fails, Grafana will fail to start back up, with an informative error as to why.